src/app/services/project-repository.service.ts

```diff
@@ -1,7 +1,7 @@
 import {Injectable} from "@angular/core";
 import {CpeApiService, iPage, iProject} from "./cpe-api.service";
 import {Page, Project, Widget} from "../cpe/model/page-data-structure";
-import {Observable, forkJoin, map, mergeMap, of, from, toArray} from "rxjs";
+import {Observable, forkJoin, map, mergeMap, of, from, toArray, tap} from "rxjs";
```

Comment:
accidental import "tap"?

Reply:
Done in https://github.com/dasch-swiss/dsp-cpe/pull/17/commits/d139bdf39a1594bf7cfcfd4d68e54ee589d5a9d3